src/coreclr/tools/Common/TypeSystem/Interop/IL/Marshaller.Aot.cs

```diff
@@ -913,17 +913,29 @@ protected override void AllocAndTransformNativeToManaged(ILCodeStream codeStream
         {
```

Comment:
Can this method be deleted? I think implementing `TransformNativeToManaged` will automatically fill in this one as well.

Reply:
Probably yes. this method do not called yet.

Reply:
Remove it.